packages/terra-form-input/tests/jest/__snapshots__/InputField.test.jsx.snap

```diff
@@ -1,5 +1,38 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`InputField should render a InputField with Placeholder 1`] = `
+<Field
+  error={null}
+  errorIcon={
+    <IconError
+      viewBox="0 0 48 48"
+      xmlns="http://www.w3.org/2000/svg"
+    />
+  }
+  help={null}
+  hideRequired={false}
+  htmlFor="test-input"
+  isInline={false}
+  isInvalid={false}
+  isLabelHidden={false}
+  label="label"
+  labelAttrs={Object {}}
+  required={false}
+  showOptional={false}
+>
+  <Input
+    disabled={false}
+    id="test-input"
+    isIncomplete={false}
+    isInvalid={false}
+    name={null}
+    placeholder="Enter The Text"
+    required={false}
+    showMpageFusionPlaceholder={true}
```

Comment:
this should not be added to Input !!

Reply:
Removed the showMpageFusionPlaceholder={true}.